cves/CVE-2014-0050.yml

```diff
@@ -173,8 +187,11 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: | 
+      This bug stems from a malicious attacker curating an HTTP request with a custom Content-Type header
```

Comment:
Nit: The bug doesn't steam from a malicious attacker doing the request, the exploit does.

Reply:
Good catch